src/precompiles/native.rs

```diff
@@ -122,12 +122,14 @@ impl Precompile for ExitToNear {
             let amount = U256::from_big_endian(&input_mut[..32]).as_u128();
             input_mut = &input_mut[32..];
 
+            // TODO: You have to charge caller's account balance for this transfer.
+
```

Comment:
What do you exactly mean? The expected workflow for this is that the user transfers his ERC20 tokens to the ERC20 contract by using its `burn()` function, which will burn the provided tokens and create a call to this precompile, which will create an appropriate promise call from the Aurora account. It's expected that the sender won't have the Near account at all, only the ETH account connected to the Aurora Web3 RPC.